### PR TITLE
Bump all packages and adapt code base

### PR DESCRIPTION
**Pull request by taniki**
I was looking for a quick way to fix https://github.com/nteract/commuter/issues/254 but it seems to be a deep issue involving dependencies.

It seems like some of current code could be remove thanks to Next advancements.

### CODE REVIEW

**Comment by captainsafia** [comment]
Thanks for submitting this PR, @taniki! Let me know when it is ready for review.

**Comment by taniki** [comment]
@captainsafia Hi! I am a bit of a noob regarding Next and React while having a good understanding of javascript. So please be merciful (:

**Comment by taniki** [comment]
It seems we can take leverage of the new [api routes](https://nextjs.org/blog/next-9#api-routes) management of Next 9 to simplify the project structure.

**Comment by captainsafia** [comment]
@taniki We can! I've actually been working with them in a side project. I want to spend some more time working with them before we migrate over. I've noticed a few weird things with them, for example, I'm having trouble configuring cors properly on the API endpoints using the method they recommend in the docs. But agree that it would be good to migrate over once the feature has been fleshed out some more.

File: pages/_document.js [hunk 1]
```diff
@@ -5,15 +5,9 @@ import flush from "styled-jsx/server";
...
-  static getInitialProps(context: Object) {
-    const renderPage = context.renderPage;
-    const { html, head, errorHtml, chunks } = renderPage();
-    const styles = flush();
-    return { html, head, errorHtml, chunks, styles };
-  }
-
-  getChildContext() {
-    return { _documentProps: this.props };
+  static async getInitialProps(ctx) {
```
Comment (captainsafia) [inline pages/_document.js]: Good clean-up work here!

File: transforms/PlotlyTransform.js [hunk 2]
```diff
@@ -25,8 +25,6 @@ PlotlyNullTransform.MIMETYPE = NULL_MIMETYPE;
 
 class PlotlyTransform extends React.Component<*> {
   props: Props;
-  getFigure: () => Object;
```
Comment (captainsafia) [inline transforms/PlotlyTransform.js]: Good catch here!

**Comment by taniki** [comment]
@captainsafia Thanks for your reviews (:

**Comment by captainsafia** [comment]
@taniki No problem. It looks like the `yarn.lock` file needs a rebase. Let me know if you need help with this.

**Comment by taniki** [comment]
@captainsafia it should alright by now

**Comment by captainsafia** [comment]
Great! Thanks again for making these fixes. Merging now.

**Pull request merged**